templates/requirements.txt.j2

```diff
@@ -1,8 +1,8 @@
-{% if requests_version is defined %}
-requests{{ '' if requests_version == 'latest' else ('==' + requests_version) }}
-{% endif%}
 docker-py{{ '' if docker_py_version == 'latest' else ('==' + docker_py_version) }}
 dockerrotate=={{ docker_rotate_version }}
 {% if docker_compose_version is not none and docker_compose_version != '' %}
 docker-compose{{ '' if docker_compose_version == 'latest' else ('==' + docker_compose_version) }}
 {% endif %}
+{% for item in docker_base_extra_python_requirements %}
```

Comment:
i don't think pip works like that... see #2 https://pip.pypa.io/en/stable/user_guide/#requirements-files

but i'm not sure order matters so i can make the change since i don't think order matters for this.